[bug] focus-trap-reactモジュールが見つからないエラー

## 📋 概要

`dev`を起動後に`http://localhost:3000`をブラウザで開くと、`focus-trap-react`モジュールが見つからないというビルドエラーが発生します。

---

## 🐛 問題点

### 発生しているエラー

```
Module not found: Can't resolve 'focus-trap-react'
./src/components/ui/Modal.tsx (4:1)
```

### エラー詳細

- **エラー発生ファイル**: `apps/frontend/src/components/ui/Modal.tsx` (4行目)
- **エラー内容**: `import FocusTrap from 'focus-trap-react';` でインポートしようとしているが、モジュールが見つからない
- **Next.jsバージョン**: 15.5.6 (outdated)

### 現在の状況

- `package.json`には`"focus-trap-react": "^11.0.4"`が依存関係に含まれている
- しかし、実際には`node_modules`にインストールされていない可能性がある

---

## 🔍 原因調査

### 推定される原因

1. **依存関係のインストール不足**: `pnpm install`が実行されていない、または不完全に実行された
2. **node_modulesの不整合**: `node_modules`が削除された、または破損している
3. **pnpm workspaceの設定問題**: workspace内の依存関係が正しく解決されていない

### 影響範囲

- **影響を受ける機能**: `Modal`コンポーネントを使用しているすべての機能
- **影響を受けるユーザー**: 開発環境でアプリケーションを起動しようとするすべての開発者

---

## 🎯 修正方針

### 修正内容

1. **依存関係の再インストール**: `pnpm install`を実行して、`focus-trap-react`を確実にインストールする
2. **動作確認**: ビルドエラーが解消され、モーダルが正常に動作することを確認する
3. **必要に応じて**: `node_modules`と`pnpm-lock.yaml`をクリーンアップして再インストール

### 代替案

- `focus-trap-react`の代わりに、フォーカストラップ機能を自前で実装する（ただし、既存のライブラリを使用する方が保守性が高い）

---

## 📝 再現手順

1. プロジェクトルートで`pnpm dev`を実行
2. ブラウザで`http://localhost:3000`を開く
3. ビルドエラーが表示される

---

## ✅ 期待される動作

修正後、以下のように動作するべき：

1. `pnpm dev`を実行してもビルドエラーが発生しない
2. `http://localhost:3000`が正常に表示される
3. `Modal`コンポーネントが正常に動作する

---

## 🔧 作業チェックリスト

- [ ] `pnpm install`を実行して依存関係をインストール
- [ ] ビルドエラーが解消されることを確認
- [ ] `http://localhost:3000`が正常に表示されることを確認
- [ ] `Modal`コンポーネントが正常に動作することを確認
- [ ] 必要に応じて`node_modules`と`pnpm-lock.yaml`をクリーンアップ

---

## 📊 影響範囲

- **影響度**: High（開発環境でアプリケーションが起動できない）
- **緊急度**: High（開発作業がブロックされる）
- **影響を受けるコンポーネント**:
  - `apps/frontend/src/components/ui/Modal.tsx`
  - `Modal`コンポーネントを使用しているすべての機能

---

## 🔗 関連ファイル

- `apps/frontend/src/components/ui/Modal.tsx`
- `apps/frontend/package.json`
- `pnpm-lock.yaml`

---

## 📖 参考資料

- [focus-trap-react Documentation](https://github.com/focus-trap/focus-trap-react)
- [Next.js Module Resolution](https://nextjs.org/docs/messages/module-not-found)

--- a/apps/frontend/Dockerfile
+++ b/apps/frontend/Dockerfile
@@ -11,7 +11,9 @@ COPY apps/frontend/package.json ./apps/frontend/
 COPY libs/types/package.json ./libs/types/
 COPY libs/utils/package.json ./libs/utils/
 
-RUN pnpm install --frozen-lockfile
+# postinstallスクリプトをスキップ（Alpine LinuxではPlaywrightの--with-depsが動作しないため）
+# 本番環境ではPlaywrightのブラウザインストールは不要
+RUN pnpm install --frozen-lockfile --ignore-scripts
 
 # ソースコードのコピーとビルド
 COPY . .

--- a/apps/frontend/Dockerfile.dev
+++ b/apps/frontend/Dockerfile.dev
@@ -13,7 +13,9 @@ COPY apps/frontend/package.json ./apps/frontend/
 COPY libs/types/package.json ./libs/types/
 COPY libs/utils/package.json ./libs/utils/
 
-RUN pnpm install --frozen-lockfile
+# postinstallスクリプトをスキップ（Alpine LinuxではPlaywrightの--with-depsが動作しないため）
+# 開発環境ではPlaywrightのブラウザインストールは不要（E2Eテストは別コンテナで実行）
+RUN pnpm install --frozen-lockfile --ignore-scripts
 
 # ソースコードのコピー
 COPY . .

